fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -273,6 +274,26 @@ object TestHelper {
 
     fun assertYoutubeAppOpens() = intended(toPackage(YOUTUBE_APP))
 
+    fun assertVoiceSearchAppOpens() {
```

Comment:
you can reuse the grantSystemPermission() method and assertNativeAppOpens() maybe